akka-projection-r2dbc/src/main/scala/akka/projection/r2dbc/internal/R2dbcOffsetStore.scala

```diff
@@ -269,6 +291,12 @@ private[projection] class R2dbcOffsetStore(
           system.executionContext))
     else None
 
+  private def scheduleNextDelete(): Unit = {
+    if (!settings.deleteInterval.isZero && !settings.deleteInterval.isNegative)
+      system.scheduler.scheduleOnce(settings.deleteInterval, () => deleteOldTimestampOffsets(), system.executionContext)
```

Comment:
Those scheduled tasks is something I want to revisit in separate PR. Feels wrong that we never stop these scheduled tasks. Probably doesn't do much due to the idle flag but anyway.

Reply:
👍🏼 good to make sure these are not continuing